docs/communications/messages/gcs-vehicles-messages.md

```diff
@@ -35,31 +35,46 @@ Sent to the vehicle to assign a job to complete a mission.
   "tid": <unsigned 32-bit integer>,
   "time": <unsigned 64-bit integer>,
   "jobType": <string>,
-  "geofence": {
-    "topLeft": [<float>, <float>],
-    "botRight": [<float>, <float>],
-    "keepOut": <boolean>
+}
+```
+
+  - **jobType : string**
+      - Job being assigned to the vehicle to in order to complete the mission. This vehicle is
+      capable of doing the job.
+
+!!! info "Requires an [acknowledgement message][] from the vehicle."
+----------------------------------------------------------------------------------------------------
+## Geofence Message
+
+Sent to the vehicle to assign a job to complete a mission.
+
+```javascript
+{
+  "type": "geofence",
+  "id": <unsigned 32-bit integer>,
+  "sid": <unsigned 32-bit integer>,
+  "tid": <unsigned 32-bit integer>,
+  "time": <unsigned 64-bit integer>,
+  "jobType": <string>,
+  "geofence" {
+    "keepOut": [<float>, <float>][],
+    "keepIn": [<float>, <float>][]
   }
 }
 ```
 
   - **jobType : string**
-      - Job being assigned to the vehicle to in order to complete the vehicle. This vehicle is
+      - Job being assigned to the vehicle to in order to complete the mission. This vehicle is
       capable of doing the job.
 
   - **geofence**
-      - **topLeft : float[]**
-          - Array of structure `[latitude, longitude]` for top left corner of geofencing.
+      - **keepOut**
```

Comment:
specify the type (eg. **topLeft : float[]**, probably for this it is [float, float][])